packages/dev-server/src/MetroDevServer.ts

```diff
@@ -171,3 +187,73 @@ function importMetroServerFromProject(projectRoot: string): typeof Metro.Server
   }
   return require(resolvedPath);
 }
+
+function replaceMiddlewareWith(
+  app: ConnectServer,
+  sourceMiddleware: HandleFunction,
+  targetMiddleware: HandleFunction
+) {
+  const index = app.stack.findIndex(middleware => middleware.handle === sourceMiddleware);
+  if (index >= 0) {
```

Comment:
Maybe use `find` since the mutation is on an object.
```js
  const somn = app.stack.find(middleware =&gt; middleware.handle === sourceMiddleware);<br class="Apple-interchange-newline">
  if (somn) {
    somn.handle = ...
  }
```